starlingcaptureapi/file_util.py

```diff
@@ -56,3 +58,51 @@ def digest_sha256(self, file_path):
                 hasher.update(byte_block)
             return hasher.hexdigest()
         # TODO: handle error (image not found, etc.)
+
+    def cidv1(self, file_path):
+        """Generates the CIDv1 of a file, as determined by ipfs add.
+
+        Args:
+            file_path: the local path to a file
+
+        Returns:
+            CIDv1 in the canonical string format
+
+        Raises:
+            Exception if errors are encountered during processing
+        """
+
+        if not os.path.exists(os.path.expanduser("~/.ipfs")):
```

Comment:
I would extract this entire `if`-block into a private helper (maybe `_maybe_init_ipfs_repo()`), so that it is easier to focus on the main job of this function when reading the code.

That also makes me think that maybe we want to have an `ipfs_tool.py` that wraps all these interactions with the IPFS CLI, the way we have a `claim_tool.py`.

(None of this is blocking for this PR.)